apps/condo/domains/miniapp/schema/B2BAppAccessRight.js

```diff
@@ -32,8 +61,28 @@ const B2BAppAccessRight = new GQLListSchema('B2BAppAccessRight', {
         delete: false,
         auth: true,
     },
+    hooks: {
```

Comment:
Why not just field hook? Since it's will only be invoked when `accessRightSet` field is changed

Reply:
This is made in a common hook because the "app" field can be changed